include/seastar/util/log.hh

```diff
@@ -286,12 +343,12 @@ public:
     /// avoid any allocations. The \arg writer will be passed a
     /// internal::log_buf::inserter_iterator that allows it to write into the log
     /// buffer directly, avoiding the use of any intermediary buffers.
-    void log(log_level level, log_writer& writer, format_info fmt = {}) noexcept {
+    void log(log_level level, log_writer& writer, format_info_t<> fmt = {}) noexcept {
```

Comment:
This overload looks dubious. What's the point of the default argument?

But let's merge the patch first, then understand this overload.

Reply:
And without the default argument, its just a special case of the template above.

Reply:
this overload is orthogonal to the fmt change. i am changing it just because i have to, as `format_info_t` is now a template class in this change. but if i have to comment on it, i'd say, it is neither used by scylla or seastar. we could deprecate and later remove it in a separate PR.

Reply:
Yes, I'm just noticed it here, it's not related to the patch.

Reply:
as a follow-up, it's dropped in #2024